rtbkit/core/banker/go_account.h

```diff
@@ -28,8 +28,8 @@ struct GoBaseAccount {
 };
 
 struct GoRouterAccount : public GoBaseAccount {
-    std::atomic<int64_t> rate;
-    std::atomic<int64_t> balance;
+    Amount rate;
+    Amount balance;
```

Comment:
Yeah it should be fine, there is lock_guards at the GoAccounts class which touched those values.